### PR TITLE
Replace `ert_shared` with `ert.shared` following upstream change in `ert`

### PR DESCRIPTION
**Pull request by asnyv**
See: https://github.com/equinor/ert/pull/3752

### CODE REVIEW

**Comment by asnyv** [comment]
should we merge prior to official release in ert @berland? (actually, the tests pass even with ert `2.37`. I haven't looked more into it though.

File: ecl2df/hook_implementations/jobs.py [hunk 1]
```diff
@@ -4,8 +4,8 @@
 from pkg_resources import resource_filename
 
 try:
-    from ert_shared.plugins.plugin_manager import hook_implementation
-    from ert_shared.plugins.plugin_response import plugin_response
+    from ert.shared.plugins.plugin_manager import hook_implementation
+    from ert.shared.plugins.plugin_response import plugin_response
 except ModuleNotFoundError:
```
Comment (berland) [inline ecl2df/hook_implementations/jobs.py:9]: (this will mask errors)

**Comment by berland** [comment]
You need to pin `ert>=2.38.0b5` in `setup.py` in order to be able to fetch a version from pypi with `ert.shared`.

File: ecl2df/hook_implementations/jobs.py (hunk 1 above)
Comment (asnyv) [reply ecl2df/hook_implementations/jobs.py:9]: ah, yes of course

**Comment by asnyv** [comment]
Know we would have to pin, therefore I was a bit surprised that it based after my quick search and replace all @berland :wink: but wasn't sure if we wanted to pin a minimum requirement prerelease, or just wait for an official ert release?

**Comment by berland** [comment]
If you merge without pinning, the ert plugins from ecl2df will stop working (for those cloning ec2ldf) until ert 2.38 is released.

File: setup.py [hunk 2]
```diff
@@ -36,7 +36,11 @@
     "sphinx-autodoc-typehints",
     "sphinx_rtd_theme",
 ]
-EXTRAS_REQUIRE = {"tests": TEST_REQUIREMENTS, "docs": DOCS_REQUIREMENTS, "ert": ["ert"]}
+EXTRAS_REQUIRE = {
+    "tests": TEST_REQUIREMENTS,
+    "docs": DOCS_REQUIREMENTS,
+    "ert": ["ert>=2.38.0-b5"],
```
Comment (berland) [inline setup.py:42]: one dash too many: https://pypi.org/project/ert/2.38.0b5/

**Review by berland — APPROVED** [review]

**Comment by anders-kiaer** [comment]
A bit late to the party here, but if I remember correctly, having a prerelease lower bound in `setup.py` will actually tell `pip` to always use the latest (pre)release of that `package`). Probably not a problem since it is an extra `ecl2df[ert]`, but we  might want to change the lower bound to official release `2.38` in `ecl2df` when `ert` is released.

**Comment by asnyv** [comment]
@anders-kiaer that is the plan 👍

**Pull request merged**